src/App.js

```diff
@@ -1,41 +1,30 @@
+import { useEffect } from "react";
 import "./App.css";
-import logo from "./logo.png";
+import { RoutesPath } from "./Routes";
+import { getVideos } from "./serverCalls/getVideos";
+import {useVideoContext} from "./context/videoContext";
 
 function App() {
+  const {dispatch} = useVideoContext();
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await getVideos();
+        if (response) {
+          dispatch({
+            type: "LOAD VIDEOS",
+            payload: response,
```

Comment:
just a suggestion - you can send just the `response.data.videos` instead of `response` to the reducer.

Reply:
Actually I am getting response.data.videos from that server call. Thats why I am just passing the response.